test/test_type_hints.py

```diff
@@ -206,17 +197,21 @@ def is_torch_mypyini(path_to_file):
         repo_rootdir = os.path.join(test_dir, '..')
         mypy_inifile = os.path.join(repo_rootdir, 'mypy.ini')
         if not (os.path.exists(mypy_inifile) and is_torch_mypyini(mypy_inifile)):
-            self.skipTest(True)
+            self.skipTest("Can't find PyTorch MyPy config file")
+
+        import numpy
+        if numpy.__version__ == '1.20.0.dev0+7af1024':
```

Comment:
This version number will change with a new nightly, I'd use `.startswith(''1.20.0.dev0')`. And we maybe should set a reminder to retest in ~4 months, otherwise the test may break once numpy 1.20.0 is released.

Reply:
Actually the breakage should show up when running `mypy` on a dev build, so the better fix would be to add `ignore_errors = True` to the `numpy` entry in `mypy.ini`

Reply:
@rgommers, actually, I want to delete this check as soon as I understand, why it fails, because it's not a `numpy` itself, but a combination of `numpy` + `hypothesis`. Following will fail the check:
```
import numpy as np
import hypothesis
from typing import Sequence

def foo(x) -> Sequence[int]:
    inputs = np.random.permutation(x)
    return inputs
```
But if unused hypothesis import is commented out it starts to pass it.

Reply:
That's weird indeed. Also be aware that the next numpy nightly will include a `py.typed` file, so more errors may show up. 

I can't find where you saw this by the way - there's no numpy nightly used in CI right?